components/tenant-mgt/org.wso2.carbon.tenant.mgt/src/main/java/org/wso2/carbon/tenant/mgt/util/TenantMgtUtil.java

```diff
@@ -319,6 +320,45 @@ public static Tenant initializeTenant(TenantInfoBean tenantInfoBean) {
         return tenant;
     }
 
+    /**
+     * @param tenantDomain domain name of the tenant.
+     * @param tenantManager tenant manager used for tenant deletion .
+     * @param tenantId tenant id.
+     * @param serverConfigurationService serverConfigurationService.
+     * @throws Exception if there is an exception during the tenant deletion.
+     */
+    public static void deleteTenant(String tenantDomain, TenantManager tenantManager, int tenantId,
```

Comment:
Why do we need to parse all of these parameters to a Util? I feel this is not a proper util method.